site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,209 @@
+| [**Attribute Annotation Mode**](/docs/manual/advanced/attribute-annotation-mode-advanced/) | Useful for adding detailed information to objects, like color, size, or other specific characteristics.        |
+| [**Annotation with Rectangles**](/docs/manual/advanced/annotation-with-rectangles/)        | Best for simple object detection where objects have a box-like shape, such as detecting windows in a building. |
+| [**Annotation with Polygons**](/docs/manual/advanced/annotation-with-polygons/)            | Suited for complex shapes in images, like outlining geographical features in maps or detailed product shapes.  |
+| [**Annotation with Polylines**](docs/manual/advanced/annotation-with-polylines/)           | Great for annotating linear objects like roads, pathways, or limbs in pose estimation.                         |
+| [**Annotation with Ellipses**](/docs/manual/advanced/annotation-with-ellipses/)            | Ideal for objects like plates, balls, or eyes, where a circular or oval annotation is needed.                  |
+| [**Annotation with Cuboids**](/docs/manual/advanced/annotation-with-cuboids/)              | Useful for 3D objects in 2D images, like boxes or furniture in room layouts.                                   |
+| [**Annotation with Skeletons**](/docs/manual/advanced/skeletons/)                          | Ideal for human pose estimation, animation, and movement analysis in sports or medical fields.                 |
+| [**Annotation with Brush Tool**](/docs/manual/advanced/annotation-with-brush-tool/)        | Perfect for intricate and detailed annotations where precision is key, such as in medical imaging.             |
+| [**Annotation with Tags**](/docs/manual/advanced/annotation-with-tags/)                    | Useful for image and video classification tasks, like identifying scenes or themes in a dataset.               |
+
+<!--lint enable maximum-line-length-->
+
+These tools make CVAT a versatile platform for a range of annotation
+needs, from basic labeling to complex, multidimensional
+tasks in advanced computer vision projects.
+
+## Automated labeling
+
+CVAT has an automated labeling features,
+enhancing the annotation process significantly,
+potentially speeding it up by up to 10 times.
+
+> **Note:**
+> For more information,
+> see [**OpenCV and AI Tools**](/docs/manual/advanced/ai-tools/)
+
+Below is a detailed table of the supported algorithms and the platforms they operate on:
+
+<!--lint disable maximum-line-length-->
+
+| Algorithm Name                                                                                          | Category   | Framework  | CPU Support | GPU Support |
+| ------------------------------------------------------------------------------------------------------- | ---------- | ---------- | ----------- | ----------- |
+| [Segment Anything](/serverless/pytorch/facebookresearch/sam/nuclio/)                                    | Interactor | PyTorch    | ✔️          | ✔️          |
```

Comment:
All these links don't work by an obvious reason. Could you please check all other links in the article?